guards/utils/filter_manager/filter_chain.py

```diff
@@ -0,0 +1,28 @@
+from .filter_list import FilterList
+
+
+class FilterChain:
+    def __init__(self, *filter_lists):
+        self.filter_lists = filter_lists or []
+
+    @property
+    def includes(self):
+        filters = []
+        for filter_list in self.filter_lists:
+            for filter in filter_list.includes:
+                filters.append(filter)
+        return filters
+
+    @property
+    def excludes(self):
+        filters = []
+        for filter_list in self.filter_lists:
+            for filter in filter_list.excludes:
```

Comment:
my sourcery helper says if all the elements from list are appended it should be extend like this `filters.extend(iter(filter_list.excludes))`

Reply:
Great, done